nuitka/build/static_src/HelpersOperationBinaryAdd.c

```diff
@@ -59,14 +62,7 @@ PyObject *BINARY_OPERATION_ADD_OBJECT_INT(PyObject *operand1, PyObject *operand2
 
     // In-lined nb_add slot from Python int type.
     if (PyInt_Check(operand1)) {
-        long a = PyInt_AS_LONG(operand1);
-        long b = PyInt_AS_LONG(operand2);
-
-        long i = (long)((unsigned long)a + b);
-
-        if ((i ^ a) >= 0 || (i ^ b) >= 0) {
-            return PyInt_FromLong(i);
-        }
+        long_var(PyObject *operand1, PyObject *operand2);
```

Comment:
See above, also when you do what I call outlining a function, don't forget to assign its return value, and consider that when you moved the return to its inside, it became no longer a return of this function, but the new one.